src/fromager/overrides.py

```diff
@@ -40,6 +41,22 @@ def _die_on_plugin_load_failure(
     raise RuntimeError(f"failed to load overrides for {ep.name}") from err
 
 
+def find_and_invoke(distname: str, method: str, default_fn: typing.Callable, **kwargs):
+    fn = find_override_method(distname, method)
+    if not fn:
+        fn = default_fn
+    return invoke(fn, **kwargs)
+
+
+def invoke(fn: typing.Callable, **kwargs):
+    sig = inspect.signature(fn)
+    for arg_name in list(kwargs):
+        if arg_name not in sig.parameters:
+            logger.warning(f"{fn.__name__} override does not take argument {arg_name}")
```

Comment:
I can never remember how much info the `__name__` property of a function includes. Is it just the name, or does it include the module, etc.? Ideally we would log the full thing, like `package_overrides.torch.download_source` so someone looking at it can go from the warning message straight to the relevant code.

Reply:
It just includes the name. Let me try adding the full the thing